pep-0517.txt

```diff
@@ -194,13 +200,11 @@ Backends which do not provide the ``prepare_metadata_for_build_wheel`` hook may
 either silently ignore the ``metadata_directory`` parameter to ``build_wheel``,
 or else raise an exception when it is set to anything other than ``None``.
 
-Wheels built by this hook should, as far as possible, be equivalent to
-first building an sdist and then building a wheel from that sdist.
-Frontends which want to protect against backends violating this expectation
-may call ``build_sdist`` (see below) first, and then call ``build_wheel``
-in the unpacked sdist. But if build_sdist fails with NotImplementedError,
-they will fall back to calling build_wheel in the source directory,
-and rely on the backend to produce an equivalent wheel.
+To ensure that wheels from different sources are built the same way, frontends
+may call ``build_sdist`` first, and then call ``build_wheel`` in the unpacked
+sdist. But if the backend indicates that it is missing some requirements for
+creating an sdist (see below), the frontend will fall back to calling
```

Comment:
Small nitpick-- I think this should be "may" instead of "will", since whether or not you do that fallback is an implementation detail of the front end.